types/require.d.ts

```diff
@@ -1 +1,2 @@
 export default function <T = any>(moduleName: string): T;
+export function has<T = any>(moduleName: string): T;
```

Comment:
Seems this should return boolean

Reply:
Doh! you are correct

Reply:
https://github.com/emberjs/ember-test-helpers/pull/851